.github/CODEOWNERS

```diff
@@ -11,6 +11,7 @@ services/purge-cache @imbstack
 services/queue @jhford
```

Comment:
Out of curiosity, how were you able to preserve the git history? It's awesome :)